saleor/payment/gateways/adyen/plugin.py

```diff
@@ -468,12 +469,24 @@ def refund_payment(
         )
         result = api_call(request, self.adyen.payment.refund)
 
+        amount = payment_information.amount
+        currency = payment_information.currency
+        msg = f"Refund for amount {amount}{currency} has been requested."
```

Comment:
Is this message for UI or just for developers who use the API? I'm asking to make sure because we shouldn't render messages from the backend in UI.

Reply:
We already have the interface to render the msg from the external sources. The event is marked as an `EXTERNAL_SERVICE_NOTIFICATION`  which we use to put some msg from the plugins/apps and show it in the order history